doc/classes/ArrayMesh.xml

```diff
@@ -172,7 +172,7 @@
 	</members>
 	<constants>
 		<constant name="NO_INDEX_ARRAY" value="-1">
-			Default value used for index_array_len when no indices are present.
+			Default value used for [code]index_array_len[/code] when no indices are present.
```

Comment:
That's a weird description as `index_array_len` is not described anywhere. Seems like this is documenting an internal implementation detail.

It looks to me like there's no real reason for this enum to be exposed in the first place. But now that it is we can't remove it in `3.x`. I guess we can just shorten to a similarly unhelpful description but somewhat less cryptic:
```suggestion
			Default value used internally when no indices are present.
```